web/source/kmwuifloat.ts

```diff
@@ -33,131 +33,134 @@ if(!window['keyman']['ui']['name']) {
 
     // Disable UI for touch devices
     if(util['isTouchDevice']()) throw '';
-    
+
     // User interface global and local variables
     keymanweb['ui'] = {};
     var ui=keymanweb['ui'];
     ui['name'] = 'float';
-    
-    ui.KeyboardSelector = null; 
-    
+
+    ui.KeyboardSelector = null;
+
     ui.outerDiv = null;     // replaces DivKeymanWeb
     ui.innerDiv = null;     // replaces Lkdiv
-    ui.oskButton = null;    // toggles OSK on or off  
+    ui.oskButton = null;    // toggles OSK on or off
     ui.kbdIcon = null;      // keyboard icon within OSK button
     ui.selecting = false;   // control focus behaviour during selection
     ui.updateList = true;   // control keyboard list updating
     ui.updateTimer = null;  // prevent unnecessary list refreshing
     ui.floatRight = false;  // align left by default
-    ui.initialized = false; // initialization flag 
+    ui.initialized = false; // initialization flag
 
     /**
      * Display or hide the OSK from the OSK icon link
-     */     
+     */
     ui.toggleOSK = function()
     {
-      let osk = keymanweb.osk;
-      keymanweb['focusLastActiveElement']();
-      if(osk && osk['show']) 
+      keymanweb['activatingUI'](true);
+      // As the Float UI uses more advanced minification, we must preserve the 'osk' field name.
+      let osk = keymanweb['osk'];
```

Comment:
I ~~hate~~ strongly dislike the fragility of this 😭 but I don't see a way around it at present...

Also, I understand why you added the comment, but it seems probably unnecessary to scatter throughout the source files given we use this pattern throughout (e.g. `keymanweb['activatingUI']`...). I think that advice should go in a coding style guide for the web code, perhaps in the [wiki style guide](https://github.com/keymanapp/keyman/wiki/Keyman-Code-Style-Guide) for now?